README.md

```diff
@@ -111,6 +113,9 @@ handling logic.
 	# Output: Serving function...
 	```
 
+	Upon starting, the framework will listen to HTTP requests at `/` and invoke your registered function
+	specified by the `FUNCTION_TARGET` environment variable (i.e. `FUNCTION_TARGET=hello`).
```

Comment:
`FUNCTION_TARGET=hello` doesn't make sense in this scenario. `HelloWorld`?

What if we do not provide this env var, like in the above example? Let's say what happens then.

Reply:
Fixed the FUNCTION_TARGET. I'm going to decline to try and document what happens if you don't specify FUNCTION_TARGET, because it's actually pretty dependent on the implementation and I can see it changing over time.